style.css

```diff
@@ -0,0 +1,473 @@
+.body {
+    max-width: 1920px;
+    margin: 0 auto;
+    background: url('pic/iswanto-arif-448003-unsplash.png') no-repeat;
+    background-size: cover;
+    overflow: visible;
+    height: 100vh;
+}
+.header {
+    width: 100%;

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Indonesia/pullRequest?prid=5674203)